packages/cli/src/openapi/response.ts

```diff
@@ -15,11 +16,31 @@ export default class Response {
         .filter((ct) => ct)
         .map((ct) => ct!.split(';')[0]);
     };
+
+    const schemata: Record<string, OpenAPIV3.SchemaObject> = {};
+
+    this.rpcRequests.forEach((rpcRequest) => {
+      const returnValue = rpcRequest.returnValue;
+      if (!returnValue) return;
+
+      if (rpcRequest.responseContentType) {
+        const mimeType = rpcRequest.responseContentType!.split(';')[0];
```

Comment:
Just an FYI: because of the null/undefined check on the previous line, the type of `rpcRequest.responseContentType` should have changed to not include `null | undefined`. The `!` shouldn't be necessary.